exampleSite/config/_default/params.toml

```diff
@@ -508,7 +508,6 @@ bundle = false
   # 相关文章推荐配置
   [page.related]
     enable = true
-    count = 5
```

Comment:
I am not sure about this change. Why do you think it's a better idea to fix recommendation count to 4?

Reply:
1. It defaults to displaying 5, but only displays 3 at the maximum page width
2. Almost no one pulls the scrollbar to browse more, the scrollbar is meaningless 
3. 4 relate items in 800px,  it will look more beautiful. 
4. This recommendation algorithm is very rough, making it meaningless for users to choose to configure the number of recommendations

Reply:
I imagine configuring the count to 2 or 3 makes sense equally. The `800px` argument does not hold on mobile, where maybe only one recommendation can fit on screen and you will need to handle the overflow with a scrollbar anyway.

Reply:
Dynamically adapt to screen width
https://github.com/HEIGE-PCloud/DoIt/blob/ba650e83f87ab9dd9069eac7ada1e581a2762a67/assets/css/_partial/_single/_related.scss#L81-L100